DetDescr/python/HcalGeometry.py

```diff
@@ -0,0 +1,58 @@
+"""Configuration for HcalGeometry"""
+
+class HcalReadoutGeometry() :
+    """Configuration for HcalGeometry for a specific geometry
+
+    Attributes
+    ----------
+    detectors_valid : array of strings
+        Regular expressions identifying which detectors are valid for this geometry
+    """
+
+    def __init__(self) :
+
+        # parameters that must align with the geometry
+        self.detectors_valid = [ ]
+        self.hcalThicknessScint = 0.
+        self.hcalWidthScint = 0.
+        self.hcalZeroLayer = []
+        self.hcalZeroStrip = []
+        self.hcalLayerThickness = []
+        self.hcalNLayers = []
+        self.hcalNStrips = []
+        self.hcalHalfTotalWidthBack = 0.
+        self.verbose = 0
+
+    def __str__(self) :
+        """Stringify this configuration class"""
+
+        s = 'HcalReadoutGeometry'
+        return s
+
+class HcalGeometry() :
+    """Container for the various geometries
+
+    """
+
+    def __init__(self):
+        self.make_v12()
+    
+    def make_v12(self) :
+        """Create the HcalGeometry with the v12 geometry parameters
+
+        Only sets parameters that must align with the gdml
```

Comment:
Please mention how you determined the numbers that are here. A first time user would be very confused, so even just a sentence or two of explanation would be helpful.

Reply:
Implemented. Added in [here](https://github.com/LDMX-Software/ldmx-sw/pull/958/commits/51e1bfab6506a789417f7ac3c85197d363f4a58c)